src/openrct2/rct1/RCT1.h

```diff
@@ -523,7 +523,10 @@ namespace OpenRCT2::RCT1
         EntertainerSnowman = 9,
         EntertainerKnight = 10,
         EntertainerAstronaut = 11,
-
+        IceCream = 12, // Unsure
```

Comment:
Rather than slapping "unsure" on these enum members, it’s better to just test importing a few SV4s with guests eating/walking with specific foods.

Reply:
While I agree, I have no way to run RCT1 right now. If you, or someone else, could help with that, we can confirm things indeed -- including the pizza entry that's already marked unsure.

Reply:
Using "Megaworld Park" which has built-in burger shops I couldn't spot any difference on guests that already had burgers at startup (I load the game and pause).
v0.4.16:
![image](https://github.com/user-attachments/assets/1c0e1207-f2d1-4e1f-ad3c-c043d49fef3f)
This PR:
![image](https://github.com/user-attachments/assets/c7e95aa5-2f01-48dd-a905-cdd1acd298f8)


As a reminder, here is how a hot dog looks:
![image](https://github.com/user-attachments/assets/4d3385e1-ae02-4068-bf73-963bf654063c)